### ISSUE TITLE
Currently testing out the repo for a Google's Speech-To-Text 

### ISSUE THREAD

**Issue by thebantz** [fostermadeco/voice-search-example-app#1]
What I have done:

- Have updated Expo SDK accordingly (v38)
- Correctly checked out config file

What I am currently stuck on:
- I wanted to ask if you ever encountered a type error for the Algolia API such as the one in the image attached!

I believe it is a great repo and would love for it to get working! 

![image](https://user-images.githubusercontent.com/35896079/85818809-866aae00-b726-11ea-9add-639dca3d9931.png)

**Comment by gstjohn** [comment]
@kellypacker Is this something we are experiencing in our application of the library after upgrading Expo?

**Comment by kellypacker** [comment]
@thebantz @gstjohn sorry for my late reply. I just updated an app to Expo 38 and during that upgrade I had to update `react-instantsearch-native` to 6.6.0: [https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/react/](https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/react/) There were a few changes that had to be made to make that update. However, I had

**Comment by kellypacker** [comment]
@thebantz I was able to update the app to Expo 38 and didn't run into that error. Originally it ran with `react-instantsearch-native` 5.4.0, but I updated it to 6.7.0 with a small change: https://github.com/fostermadeco/voice-search-example-app/commit/96e4276867e1785f8da7b7ade77c5a25c61ac52b

I'm sure you have moved on with your life, but let me know if you run into anything else. Again sorry for my late reply!
